src/components/SearchResultList/SearchResultModalForm.tsx

```diff
@@ -0,0 +1,149 @@
+'use client';
+
+import * as React from 'react';
+import Modal from 'react-modal';
+
+import { Formik } from 'formik';
+
+import { MODAL_DEFAULT_STYLES } from '@/constants/constants';
+import { SearchResultFilter } from '@/types/query';
+
+interface InjectedProps {
+  isOpenModal: boolean;
+
+  openModal: () => void;
+  closeModal: () => void;
+  onApply: (queryParams: SearchResultFilter) => void;
+}
+
+const SearchResultModalForm = (props: InjectedProps) => {
+  const { closeModal, isOpenModal, onApply } = props;
+  return (
+    <div data-testid="search-result-filter-modal">
+      <Modal
+        isOpen={isOpenModal}
+        onRequestClose={closeModal}
+        style={MODAL_DEFAULT_STYLES}
+        ariaHideApp={false}
+      >
+        <div className="ReactModalPortal__head">
```

Comment:
Why is the class name casing different? And not:

```suggestion
        <div className="search-result-filter-modal__head">
```

The modal component (https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/103/files#diff-a128ba64d2993a021e074681d30d6e58fcdee69d7054604d91d63d38309ce520) should not use `ReactModalPortal` as the block element (as this comes from the library). Think of combining class names:

```react
<div data-testid="modal search-result-filter-modal">
   <Modal
        isOpen={isOpenModal}
        onRequestClose={closeModal}
        style={MODAL_DEFAULT_STYLES}
        ariaHideApp={false}
      >

  <div className="modal__head">

  #...
</div>
```

`modal` is the shared component that can be used in multiple page, while `search-result-filter-modal` can be used to target this very specific modal

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/103/commits/e4d939d6e32197ea100fa7403e6fcf3f6d019cb6. Remove the modal using new implementaiton.